src/topp/MzTabExporter.cpp

```diff
@@ -369,16 +367,16 @@ namespace OpenMS
         row.mass_to_charge = MzTabDouble(f.getMZ());
         MzTabDoubleList rt_list;
         vector<MzTabDouble> rts;
-        rts.push_back(MzTabDouble(f.getRT()));
+        rts.emplace_back(MzTabDouble(f.getRT()));
```

Comment:
the point of `emplace_back` is to avoid the move, yet you are constructing a temporary (so this code does not actually run faster).
You should be using `rts.emplace_back(f.getRT());` to get the performance benefit. Or did I miss the point here?
Applies also for other replacements below (especially make_pair).

Reply:
True, is used `emplace_back wrongly here.